canonical-web-and-design/maas-ui
# feat(ui): update LXD and virsh table content

## Done

- Updated LXD table to be grouped by server address
- Update table content to match [wireframe](https://app.zeplin.io/project/601454d5d1141e1b6ea271b7/screen/602aaa0b9018077e8e566d4e) (just missing LXD version for now)
- Made LXD and virsh tables responsive
- Added scss mixin for truncating a table row border

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the KVM list and check that the tables match the [wireframe](https://app.zeplin.io/project/601454d5d1141e1b6ea271b7/screen/602aaa0b9018077e8e566d4e)
- Check that the tables are reasonably responsive

## Fixes

Fixes #2262
